gender_novels/novel.py

```diff
@@ -119,6 +119,30 @@ def __str__(self):
         name = self.filename[0:len(self.filename)-4]
         return name
 
+    def __repr__(self):
```

Comment:
For the time being, it looks like __repr__ returns exactly the same thing as __str__ in which case, you could be lazy 
and make the code just 
```
return self.__str__()
```
Now, my understanding is that that would be perfectly fine but also that the expectation for repr is that it should return a bit more useful information for a developer. Though I should say that I haven't written a __repr__ function before. One possible way of doing that would be to have repr also show that you are dealing with a Novel object: 
```
        name = self.filename[0:len(self.filename) - 4]
        return f'Novel ({name})'
```

Reply:
Sometimes people put this in angle brackets, so

```
return f'<Novel ({name})>'
```
would match convention.  Generally we either put `<ClassName someinfo>` or if it's possible to recreate the object easily then `Novel(name)` (but I don't think this is the case).